Userland/Libraries/LibCpp/Parser.cpp

```diff
@@ -930,7 +925,7 @@ Optional<size_t> Parser::index_of_token_at(Position pos) const
 void Parser::print_tokens() const
 {
     for (auto& token : m_tokens) {
-        dbgln("{}", token.to_string());
+        outln("{}", token.to_string());
```

Comment:
Is that change intentional?

Reply:
yep, the command line tool `cpp-parser` uses this function for its `-T` option (print tokens).
So we want this function to output to stdout rather than the debug console.